annotation/pom.xml

```diff
@@ -12,6 +12,14 @@
 
   <name>SmallRye Common: Annotations</name>
 
+  <dependencies>
+    <dependency>
+      <groupId>jakarta.enterprise</groupId>
+      <artifactId>jakarta.enterprise.cdi-api</artifactId>
+      <scope>provided</scope>
```

Comment:
Not exactly sure about this line, but I guess we don't want to drag full CDI API with SRye Common Annotations. Any ideas?

Reply:
I think `provided` is appropriate for this general situation.  I'm a little concerned that this brings in a tree of dependencies though.  Could we add exclusions to keep it minimal?  Also, are we OK with inheriting the version from the SmallRye parent?

Reply:
I can prune it down to just `jakarta.inject` and `jakarta.enterprise.cdi-api` itself, though the tree of dependencies of `jakarta.enterprise.cdi-api` is rather small. But better to be explicit in this case, I guess.

Reply:
I did this. Inheriting the CDI API version from SmallRye Parent should be fine, we really only need the `@Qualifier` annotation and the `AnnotationLiteral` superclass, both of which should be very stable.